features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -1,33 +1,56 @@
 # Supporting PSA in Mbed OS
 
-This document is still a work in progress. It describes the process of adding new PSA targets to Mbed OS. It also outlines the new directory structure created to support PSA on single v7-M, dual v7-M and single v8-M.
+This document is still a work in progress. It describes the process of adding new PSA targets to Mbed OS and building [TF-M](https://git.trustedfirmware.org/trusted-firmware-m.git/tree/?h=feature-twincpu) for v8-M and dual-core targets. It also outlines the new directory structure created to support PSA on all targets.
 
-## TODO
-* Adding dual v7-M targets
+*Glossary*  
+PSA - Platform Security Architecture  
+TF-M - Trusted Firmware M  
+SPE - Secure Processing Environment  
+NSPE - Non-Secure Processing Environment
+
+# TODO
 * Adding v8-M targets
-* Building TF-M
 * Running PSA compliance tests
 
-## Adding new PSA targets
-To help with the creation of new PSA targets, couple of generic PSA targets have been added to `targets/targets.json`.
+# Adding new PSA targets
```

Comment:
This section feels like it should be part of the porting guide in our handbook

Reply:
@bulislaw Yes. The idea is to capture everything about PSA support in Mbed OS in a document (supporting_psa_in_mbed-os.md) and when we are ready to merge the feature branch to master, update the porting guide as well the docs with the information already in supporting_psa_in_mbed-os.md.